cloudinit/sources/DataSourceEc2.py

```diff
@@ -28,6 +28,10 @@
 STRICT_ID_PATH = ("datasource", "Ec2", "strict_id")
 STRICT_ID_DEFAULT = "warn"
 
+API_TOKEN_ROUTE = 'latest/api/token'
+API_TOKEN_DISABLED = '_ec2_disable_api_token'
+AWS_TOKEN_TTL_SECONDS = '21600'
```

Comment:
@blackboxsw and @raharper As I was testing this, I could see this token was being printed out in cloud-init.logs.
Also, Since the ttl is 6 hours, I could use the token and make calls to ec2 metadata service. Do you think its is right idea to :

Make sure the token value is not printed in the logs?
If it has to be printed, Should this ttl value be low enough to make sure the token become unusable soon.